vagrant/provision/roles/hubot/files/ninjabot/scripts/update.coffee

```diff
@@ -31,35 +32,34 @@ module.exports = (robot) ->
     switch host
       when 'monitoring'
         server='monitoring-node'
-        command='sh ./provision.sh --limit=#{server} monitoringserver.yml'
```

Comment:
😁 need to be `./provision.sh -local --limit...` without this, there is an rsync which is not triggered in provision.sh, and we just reprovision the old version of the ansible playbook 😁